lib/Doctrine/Migrations/AbstractMigration.php

```diff
@@ -126,9 +126,10 @@ public function down(Schema $schema): void
     protected function addSql(
         string $sql,
         array $params = [],
-        array $types = []
+        array $types = [],
+        bool $executeAsStatement = false
     ): void {
-        $this->plannedSql[] = new Query($sql, $params, $types);
+        $this->plannedSql[] = new Query($sql, $params, $types, $executeAsStatement);
     }
```

Comment:
I think boolean flags are a code smell. Let's add a new method called `addSqlStatement()` instead?

Reply:
You mean, like a static constructor method ob `Query`?

Reply:
No, I mean like a new method next to `addSql`, called `addSqlStatement`. 

```php
protected function addSqlStatement(
        string $sql,
        array $params = [],
        array $types = [],
): void {
    $this->plannedSql[] = new Query($sql, $params, $types, true);
}